src/svg-quirks-mode/fonts/LICENSE.txt

```diff
@@ -0,0 +1,202 @@
+
+                                 Apache License
+                           Version 2.0, January 2004
+                        http://www.apache.org/licenses/
```

Comment:
@thisandagain Where should that go? I could import scratch-flash as a dependency, or do you want a new repo scratch-fonts?